### ISSUE TITLE
Support of newer Symfony Versions?

### ISSUE THREAD

**Issue by Beleggrodion** [GeniusesOfSymfony/WebSocketBundle#486]
Is it planned to support newer Symfony Versions?

Because the current LTS is 6.4.x and the complete new version is 7.1.x and the current supported version 5.3.x has already EOL and 5.4.x is near EOL https://symfony.com/releases

**Comment by vladimirgolub** [comment]
Yes, Symfony 6.4.x is the current LTS, and Symfony 7.1.x has been released as the latest stable version. Since Symfony 5.3.x has already reached EOL and 5.4.x is nearing EOL, it is becoming increasingly important to support newer Symfony versions in the GeniusesOfSymfony/WebSocketBundle.

Additionally, many users strongly rely on this bundle for their projects and require compatibility with newer Symfony versions to ensure long-term support, stability, and the ability to leverage the latest Symfony features.

**Comment by BUGOVER** [comment]
https://github.com/BabDev/WebSocketBundle
